pkg/api/k8s_versions.go

```diff
@@ -468,6 +468,7 @@ func getK8sVersionComponents(version string, overrides map[string]string) map[st
 	case "1.11":
```

Comment:
how about 1.12, 1.13 and 1.14?
Please also ensure the images exists under docker account msazurestackdocker

Reply:
Only v1.12.7 is in docker hub.
However, I would rather have a separate PR for newer minor versions once we have a successful 1.11.X deployment on a disconnected azure stack

Reply:
This seems to be working, adding 1.12